example/bench/recipe.cpp

```diff
@@ -0,0 +1,54 @@
+#include <random>
+
+#include <benchmark_cell.hpp>
+#include <common_types.hpp>
+#include <time_sequence.hpp>
+
+#include "recipe.hpp"
+
+cell_size_type bench_recipe::num_cells() const {
+    return params_.num_cells;
+}
+
+arb::util::unique_any bench_recipe::get_cell_description(cell_gid_type gid) const {
+    using RNG = std::mt19937_64;
```

Comment:
Why all capital?

Reply:
Good question. I have used this pattern in a few places, and it was originally cut and paste from code where `RNG` was a template parameter.
I replaced it with `rng_type`.